samples/wifi/throughput/README.rst

```diff
@@ -0,0 +1,308 @@
+.. _wifi_throughput_sample:
+
+Wi-Fi: Throughput
+#################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Throughput sample demonstrates how to measure the throughputs for Nordic Semiconductor's Wi-Fi® chipsets.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+The sample demonstrates how to measure network performance using **zperf** tool in different profiles.
```

Comment:
Can we add a link to the ZPERF tool in Zephyr Documentation?

Reply:
also: network throughput performance? or simply network throughput?

Reply:
also, what are the different "profiles"? I'd like a more detailed description here, e.g. "different Wi-Fi stack configuration profiles"

Reply:
Proposal " The sample demonstrates how to measure the network throughput of a Nordic Wi-Fi enabled platform using **zperf** tool under the different Wi-Fi stack configuration profiles"